.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 name: Continuous Integration
 
-on: [push, pull_request]
+on: push
```

Comment:
Does the CI run on every `push` event even when there is no pull request?

Reply:
Yes, it runs on every `git push` to remote.